pkg/apis/kops/validation/gce.go

```diff
@@ -44,15 +46,32 @@ func gceValidateCluster(c *kops.Cluster) field.ErrorList {
 		}
 	}
 
+	gcpAliasRange := c.Spec.CloudConfig.GCPAliasRange
+	if !gceValidateAliasRange(gcpAliasRange) {
+		f := fieldSpec.Child("CloudConfig")
+		allErrs = append(allErrs, field.Invalid(f.Child("gcpAliasRange"), gcpAliasRange, "must match /2[4-8]"))
+	}
+
 	return allErrs
 }
 
 func gceValidateInstanceGroup(ig *kops.InstanceGroup, cloud gce.GCECloud) field.ErrorList {
 	allErrs := field.ErrorList{}
 
+	fieldSpec := field.NewPath("spec")
+
 	if ig.Spec.GCPProvisioningModel != nil {
-		fieldSpec := field.NewPath("spec")
 		allErrs = append(allErrs, IsValidValue(fieldSpec.Child("gcpProvisioningModel"), ig.Spec.GCPProvisioningModel, []string{"STANDARD", "SPOT"})...)
 	}
+
+	gcpAliasRange := ig.Spec.GCPAliasRange
+	if !gceValidateAliasRange(gcpAliasRange) {
+		allErrs = append(allErrs, field.Invalid(fieldSpec.Child("gcpAliasRange"), gcpAliasRange, "must match /2[4-8]"))
+	}
 	return allErrs
 }
```

Comment:
Note, validation doesn't seem to run when doing a `kops replace -f` but it does for `kops create` and `kops update` .  This seems like a bug outside the scope of this change.